src/ProjectDataProvider.ts

```diff
@@ -316,7 +316,8 @@ export module ProjectDataProvider {
     return new Promise((resolve, reject) => {
       const outputChannelDep = isOutputChannelActivated();
       outputChannelDep.clearOutputChannel();
-      let vscodeRootpath = item.replace('go.mod', '');
+      // Remove trailling '/' or '\' in order to provide perfect folder path to gomanifest
+      let vscodeRootpath = item.replace(/[\/|\\]$/, '');
```

Comment:
probably you may need to do same treatment as LSP using url package.

Reply:
I did that but same trick did not work here as `item` passed to this function is a folder path ending with one extra '\', when we use `dirname()` it removes last directory, which becomes invalid path further.
If you trace this flow, `item` we are passing is fullManifestPath.split("go.mod")[0], example if go.mod is at /home/user/sample/goprject/go.mod => item will be `/home/user/sample/goproject/` and on applying `dirname()` it will become `/home/user/sample` (it looses `goproject` directory).

Reply:
@dgpatelgit , I'm bit confused now :) What does `item` carries? can you post some sample values?

nit: Please rename `item` to something meaningful :)

Reply:
`item` == `/home/user/sample/goproject/`  if you have go.mod at `/home/user/sample/goproject/go.mod`

Reply:
So, it is a path to `go.mod` root directory., what would the value incase of windows?

Reply:
in case of windows `item` == `c:\Users\sample\goproject\`

Reply:
IMHO, the right fix would be to use `dirname` in https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/blob/0a334d9f9967ffb1918a67a7e0598340362975c9/src/stackanalysismodule.ts#L134 instead of split on the manifest name. It might look bit of an extra thing for issue, but it will be a clean solution.

Reply:
I had this as first solution, but then thought we are doing bit extra to address the issue. But now we both have similar opinion, let me change and avoid this split. I don't know when this was added, initial implementation of golang did not have this split. We used to replace 'go.mod' with '' for `item` variable.

Reply:
AFAIK, This is in the same state, nothing changed anything in this regard. Anyways, `git blame` is your fried if you wanna dig.